src/imbi/endpoints/blueprints.py

```diff
@@ -0,0 +1,206 @@
+"""Blueprint CRUD endpoints"""
+
+import logging
+import typing
+
+import fastapi
+import neo4j
+
+from imbi import models
+from imbi import neo4j as imbi_neo4j
+
+LOGGER = logging.getLogger(__name__)
+
+blueprint_router = fastapi.APIRouter(prefix='/blueprints', tags=['blueprints'])
+
+
+@blueprint_router.post('/', response_model=models.Blueprint, status_code=201)
+async def create_blueprint(blueprint: models.Blueprint) -> models.Blueprint:
+    """Create a new blueprint.
+
+    Creates a blueprint node in the graph database. The slug will be
+    auto-generated from the name if not provided.
+
+    Returns:
+        The created blueprint with round-trip values from the database
+
+    Raises:
+        409: Blueprint with same name and type already exists
+    """
+    try:
+        node = await imbi_neo4j.create_node(blueprint)
+        return models.Blueprint.model_validate(dict(node))
```

Comment:
move this to neo4j/__init__.py's create_node.  This should just be `return await ...`